internal/pkg/util/fs/helper_linux.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
```

Comment:
This looks like this file was created in 2019

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
```